tools/Makefile

```diff
@@ -72,6 +72,10 @@ sentencepiece.done:
 moses:
 	git clone https://github.com/moses-smt/mosesdecoder.git moses
 
+nara_wpe: venv
+	. venv/bin/activate; pip install git+https://github.com/fgnt/nara_wpe.git
```

Comment:
Why not using `pip install nara_wpe`?

Reply:
"pip install nara_wpe" installs an old version, that is why I give the path